src/applications/bmqtool/m_bmqtool_parameters.h

```diff
@@ -705,6 +716,12 @@ inline const bsl::vector<Subscription>& Parameters::subscriptions() const
     return d_subscriptions;
 }
 
+inline
+const bsl::string &Parameters::autoIncrementedField() const
```

Comment:
Making formatter happy
```suggestion
inline const bsl::string &Parameters::autoIncrementedField() const
```